### PR TITLE
Azure deploy

### PR DESCRIPTION
**Pull request by codemillmatt**

### CODE REVIEW

File: azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,403 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "functionAppName": {
+            "type": "string",
+            "metadata": {
+                "description": "Function App Nam. Must be globally unique."
```
Comment (Redth) [inline azuredeploy.json:8]:
```suggestion
                "description": "Function App Name. Must be globally unique."
```
Typo

File: azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+            "type":"string",
+            "metadata": {
+                "description": "the prefix that will be used in the naming of containers in Blob storage for each region your app/server supports. The default is 'region'"
+            },
+            "defaultValue":"region"
+        },
+        "supportedRegions": {
+            "type": "string",
+            "metadata": {
+                "description": "Two character ISO 3166 country codes of regions the server supports. Semi-colon delimited."
```
Comment (Redth) [inline azuredeploy.json:38]:
```suggestion
                "description": "3 digit Mobile Country Codes (https://en.wikipedia.org/wiki/Mobile_country_code) for the regions the server supports. Semi-colon delimited."
```

File: azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+            },
+            "defaultValue": "us;ca"
```
Comment (Redth) [inline azuredeploy.json:40]:
```suggestion
            "defaultValue": "302;310;311;312;313;314;315;316"            
```

File: azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,403 @@
...
+            },            
+            "minLength": 3,
+            "maxLength": 24
+        },
+        "blobContainerPrefix": {
+            "type":"string",
+            "metadata": {
+                "description": "the prefix that will be used in the naming of containers in Blob storage for each region your app/server supports. The default is 'region'"
+            },
+            "defaultValue":"region"
```
Comment (Redth) [inline azuredeploy.json:33]:
```suggestion
            "defaultValue":"region-"
```
Just want to be consistent with what the client app is using as the default.

File: azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,403 @@
...
+            "metadata": {
+                "description": "Hosting Service Name. Globally unique."
+            },            
+            "minLength": 3,
+            "maxLength": 24
+        },
+        "blobContainerPrefix": {
+            "type":"string",
+            "metadata": {
+                "description": "the prefix that will be used in the naming of containers in Blob storage for each region your app/server supports. The default is 'region'"
```
Comment (Redth) [inline azuredeploy.json:31]:
```suggestion
                "description": "the prefix that will be used in the naming of containers in Blob storage for each region your app/server supports. The default is 'region-'"
```

File: azuredeploy.json (hunk 4 above)
Comment (codemillmatt) [reply azuredeploy.json:33]: The hyphen gets appended later on. This way they don't have to worry about it at all.

File: azuredeploy.json (hunk 5 above)
Comment (codemillmatt) [reply azuredeploy.json:31]: Kinda the same comment as before... the hyphen will get appended on during the deployment process, so we don't have to rely on anybody to remember to enter it.

**Comment by codemillmatt** [comment]
Closing this PR - have some work to do around KeyVault 😢

**Pull request closed without merging**